source/github/repo.go

```diff
@@ -16,6 +17,20 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// ConfigBackoff is a wrapper for Config that will retry five times if the function fails to retrieve the yaml/yml file.
+func (c *client) ConfigBackoff(u *library.User, org, name, ref string) (data []byte, err error) {
+	for i := 0; i < 5; i++ {
```

Comment:
It's not necessary, but having a variable for retry limit and sleep time would make it cleaner and easier to update if we want to tweak those values